design-system/materials/internals/definition.yaml

```diff
@@ -73,7 +78,41 @@ choiceGroupsByTheme:
       label: Font sizes
       prefix: font-size
       choices:
-        font-size-m: '1rem'
+        font-size-10: '0.75rem'
+        font-size-20: '0.875rem'
+        font-size-30: '1rem'
+        font-size-40: '1.125rem'
+        font-size-50: '1.25rem'
+        font-size-60: '1.5rem'
+        # These font sizes will be removed when we replace the default theme.
+        font-size-15: '0.813rem'
+        font-size-70: '2rem'
+
+    fontWeights:
+      label: Font weights
+      prefix: font-weight
+      choices:
+        font-weight-300: '300'
+        font-weight-400: '400'
+        font-weight-500: '500'
+        font-weight-600: '600'
```

Comment:
I don't know but to me that made most sense because we already use a number scale and this way it's most straightforward which value are we aiming for

Reply:
Bear in mind we currently are using three different weights (300, 400 and 700) but in the new theme we will be using four (400, 500, 600 and 700).
If we want to use a more semantic name for the tokens, we need to mind all the options.

In the past I used light, normal and bold, but in this context we will have two values between normal and bold.
We can tailor the naming to match our current scale, but I don't see it as a positive change for us in the future.

Personally I would rather keep using the numbered naming.